libs/elasticsearch/scripts/run_unasync.py

```diff
@@ -111,7 +111,7 @@ def main(check=False):
     for dir in source_dirs:
         output_dir = f"{dir[0]}_sync_check/" if check else dir[1]
         subprocess.check_call(["ruff", "format", "--target-version=py38", output_dir])
-        subprocess.check_call(["isort", output_dir])
+        subprocess.check_call(["ruff", "check", "--fix", "--select", "I", output_dir])
```

Comment:
We already have `--select I` in pyproject.toml.

Reply:
I'm just repeating what's in the Makefile:

https://github.com/langchain-ai/langchain-elastic/blob/686575ca13bd9b0936d38a3efed4803b47711f6e/libs/elasticsearch/Makefile#L30-L38

Reply:
I suspect it's not needed! But not familiar with ruff to tell.